azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -376,6 +376,7 @@ internal open class CallCompositeActivity : AppCompatActivity() {
                 store.end()
                 callingMiddlewareActionHandler.dispose()
                 videoViewManager.destroy()
+                callStateHandler.onCompositeExit()
```

Comment:
This looks duplicate to `CompositeExitManager.notifyCompositeExit()`.
Why do we need to notify Contoso about call state changed event before exit. Consoto should have received it on actual call state change.
If there is a race condition, and, for instance `Disconnected` state changed event is not raised before composite is dismissed, then Contoso should rely on the `onDismissed` event.